src/plugins/auto/src/plugin.cpp

```diff
@@ -411,7 +395,8 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
         autoSContext->_needPerfCounters = true;
     }
     autoSContext->_modelPriority = MapPriorityValues(loadConfig.get_property(ov::hint::model_priority));
-    autoSContext->_batchingDisabled = !(loadConfig.get_property(ov::hint::allow_auto_batching));
+    auto itorConfigAutoBatch = config.find(ov::hint::allow_auto_batching.name());
+    autoSContext->_batchingDisabled = itorConfigAutoBatch != config.end() && itorConfigAutoBatch->second == "false";
```

Comment:
if directly check from config, need to consider 1.0 API, which value is "NO"
or we can check from loadConfig, which internally been wrapped to 2.0

Reply:
Modified, please review.